zboxcore/sdk/sync.go

```diff
@@ -29,9 +29,10 @@ const (
 )
 
 type fileInfo struct {
```

Comment:
Actual Size is already added to fileRef
https://github.com/0chain/gosdk/commit/a7eeb212ef107e889f80bdf90c274b9c8f41019f